orbit/changes/18130-linux-end-user-email

```diff
@@ -0,0 +1 @@
+- Added support for `--end-user-email` option when building fleetd Linux packages.
```

Comment:
This PR should have two changes file, this one (`orbit/changes`) and in `changes/`.